README.md

```diff
@@ -238,6 +238,171 @@ To run the examples in a browser using WebAssembly, you can use [wasm-server-run
+
+- **bevy_pkv doesn't do automatic management**
+
+With [bevy_pkv](https://github.com/johanhelsing/bevy_pkv), modifying the object and making the changes persistent is always two different steps.
+
+```rust
+fn modify_key_bindings(mut pkv: ResMut<PkvStore>) {
+  let mut key_bindings = pkv.get::<KeyBindings>("key-bindings");
+  key_bindings.crouch = KeyCode::LControl;
+  pkv.set("key-bindings", &key_bindings)
+}
+```
+
+[bevy-persistent](https://github.com/umut-sahin/bevy-persistent/) on the other hand provides APIs to automate this process (see [Modification](#modification)).
+
+- **bevy_pkv makes external edits very hard (in native apps)**
+
+[bevy-persistent](https://github.com/umut-sahin/bevy-persistent/) stores each persistent object as a separate file in the specified format. If a textual format is used, the object becomes extremely easy to edit! The example in [Creation](#creation) will create `key-bindings.toml` in the specified location with the following contents:
+
+```toml
+jump = "Space"
+crouch = "C"
+```
+
+You can easily change this file to:
+
+```toml
+jump = "Space"
+crouch = "LControl"
+```
+
+And it'd work perfectly in the next run!
+
+[bevy_pkv](https://github.com/johanhelsing/bevy_pkv) on the other hand stores the objects in a single database in a binary format, which is more efficient but not suitable external modification.
+
+- **bevy_pkv makes storing objects in different locations very hard (in native apps)**
+
+This one is kinda like the one above, but it's more about how things are structured to satisfy your needs! For example, you may want to synchronize user saves through [Steam](https://store.steampowered.com/) but not user settings as they need to be different across machines.
+
+This scenario is not easy to achieve with [bevy_pkv](https://github.com/johanhelsing/bevy_pkv) because of the same reason as above, but it's very easy with [bevy-persistent](https://github.com/umut-sahin/bevy-persistent/) as you can structure your persistent objects however you see fit in the filesystem!
+
+- **bevy_pkv only supports JSON and Binary storage formats**
+
+[bevy-persistent](https://github.com/umut-sahin/bevy-persistent/) supports a wide variety of storage formats! If you're porting a game from another engine to [Bevy](https://bevyengine.org/) you might be able to just define the structure of some objects (e.g., settings, saves) and let [bevy-persistent](https://github.com/umut-sahin/bevy-persistent/) handle the rest!
+
+With [bevy_pkv](https://github.com/johanhelsing/bevy_pkv) however, you need to create a transformation layer as it's not directly compatible with any of the widely used formats.
+
+- **bevy_pkv is not type safe**
+
+[bevy-persistent](https://github.com/umut-sahin/bevy-persistent/) integrates with type system of [Bevy](https://bevyengine.org/). Types of persistent resources are specified in system definitions and everything is handled by [Bevy](https://bevyengine.org/).
```

Comment:
Are there any differences if the model changes? Are we both using serde?

Reply:
This wasn't about (de)serialization, it was more about `PkvStore` being generic in terms of types.

You can call these back to back `pkv.get::<Settings>("same_key")` and `pkv.get::<AutoSave>("same_key")` and you'd get error in one of the cases at runtime, whereas you'd get the error during setup with bevy-persistent.

Reply:
Also, you can't mistype a key as it's in the type system.